src/bookshelf/rebar.config

```diff
@@ -2,45 +2,103 @@
 %% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
 %% ex: ts=4 sw=4 ft=erlang et
 
-{require_otp_vsn, "18.*|19.*"}.
+
 
 {erl_dep_retries, 10}.
 
+
+
+
+{require_otp_vsn, "18.*|19.*"}.
+
 {deps, [
-        {lager,".*",
-         {git,"https://github.com/basho/lager", {tag, "2.1.1"}}},
-        {meck, ".*",
-         {git, "https://github.com/eproxus/meck.git", {tag, "0.8.4"}}},
-        {chef_secrets, ".*",
-         {git, "https://github.com/chef/chef_secrets", {branch, "master"}}},
-        {iso8601, ".*",
-         {git, "https://github.com/erlsci/iso8601", {tag, "1.2.3"}}},
-        {opscoderl_wm, ".*",
-         {git, "https://github.com/chef/opscoderl_wm.git", {branch,"master"}}},
-        {erlsom, ".*",
-         {git, "https://github.com/chef/erlsom",
-          {branch, "integer_long_string_probs"}}},
-        {mochiweb, ".*",
-         {git, "git://github.com/mochi/mochiweb", {tag, "v2.12.2"}}},
-        {erlware_commons, ".*",
-         {git, "https://github.com/erlware/erlware_commons", {branch, "master"}}},
-        {cf, "",
-         {git, "https://github.com/project-fifo/cf", {branch, "master"}}},
-        {mixer, ".*",
-          {git, "https://github.com/chef/mixer", {tag, "0.1.1"}}},
-        {mini_s3, ".*",
-         {git, "https://github.com/chef/mini_s3", {branch, "master"}}},
-        {sqerl, ".*",
-         {git, "https://github.com/chef/sqerl", {branch, "master"}}},
-        {envy, ".*",
-         {git, "https://github.com/markan/envy", {branch, "master"}}},
-        {sync, ".*",
-         {git, "https://github.com/rustyio/sync", {branch, "master"}}},
-        {eper, ".*",
-         {git, "https://github.com/massemanet/eper", {branch, "master"}}},
-        {observer_cli, ".*",
-         {git, "https://github.com/zhongwencool/observer_cli", {branch, "master"}}}
-       ]}.
+    %% lager has to come first since we use its parse transform
+    {lager, ".*",
+        {git, "https://github.com/basho/lager.git",                     {branch, "master"}}},
+%   {bcrypt, ".*",
+%       {git, "https://github.com/smarkets/erlang-bcrypt.git",          {branch, "master"}}},
+    {cf, "",
+        {git, "https://github.com/project-fifo/cf.git",                 {branch, "master"}}},
+%   {chef_authn, ".*",
+%       {git, "https://github.com/chef/chef_authn.git",                 {branch, "master"}}},
+%   {chef_reindex, ".*",
+%       {git, "https://github.com/opscode/chef_reindex.git",            {branch, "master"}}},
```

Comment:
We should get rid of the commented dependencies if we do not need them.

Reply:
Applicable in other places that have commented dependencies.

Reply:
i'll show you why i have the comments there when we can zoom.